papermill/utils.py

```diff
@@ -10,6 +10,48 @@
 logger = logging.getLogger('papermill.utils')
 
 
+def any_tagged_cell(nb, tag):
+    """Whether the notebook contains at least one cell tagged ``tag``?
+
+    Parameters
+    ----------
+    nb : nbformat.NotebookNode
+        The notebook to introspect
+    tag : str
+        The tag to look for
+
+    Returns
+    -------
+    bool
+        Whether the notebook contains a cell tagged ``tag``?
+    """
+    return any([tag in cell.metadata.tags for cell in nb.cells])
+
+
+def find_first_tagged_cell_index(nb, tag):
```

Comment:
We could probably use these in our [papermill wrapper](https://github.com/viaduct-ai/vml-py/blob/master/src/vml/papermill/papermill_sparkmagic_wrapper.py)

Reply:
truth